data-prepper-plugins/otel-trace-raw-processor/src/main/java/com/amazon/dataprepper/plugins/processor/oteltrace/OTelTraceRawProcessor.java

```diff
@@ -39,14 +41,14 @@
     private static final BigDecimal SEC_TO_MILLIS = new BigDecimal(1_000);
 
     private static final Logger log = LoggerFactory.getLogger(OTelTraceRawProcessor.class);
+    private final Counter spanCounter;
+    private final Counter resourceSpanCounter;
 
     //TODO: https://github.com/opendistro-for-elasticsearch/simple-ingest-transformation-utility-pipeline/issues/66
     public OTelTraceRawProcessor(final PluginSetting pluginSetting) {
-        this();
-    }
-
-    private OTelTraceRawProcessor() {
-
+        super(pluginSetting);
+        spanCounter = this.pluginMetrics.counter("spanCounter");
+        resourceSpanCounter = this.pluginMetrics.counter("resourceSpanCounter");
```

Comment:
Same here. If a user sees a metric called Pipeline.OtelTraceRawProcessor.resourceSpanCounter, they will think its counting the number of resource spans that are found

Reply:
We might want totalErrors as well.